### ISSUE TITLE
TerraSpin JAR requires installation to same pod as Halyard

### ISSUE THREAD

**Issue by hodnett** [OpsMx/Terraform-spinnaker#1]
This design paradigm makes it difficult to accomplish either of these scenarios:

1. Multiple Terraspin containers or pods accessed via a load balancer, allowing Spinnaker to execute multiple pipelines in parallel.
2. Multiple Spinnaker clusters, each with their own dedicated Terraspin container, yet all are created, managed and destroyed by the same Halyard.

We recommend updating Terraspin to _optionally_ permit reading from static configuration files that can be baked into its container.

**Comment by markyjackson-taulia** [comment]
@hodnett Are you still interested in this? I ask because I can certainly look at it but if you're not then I can focus elsewhere.

**Comment by markyjackson-taulia** [comment]
Going to close this out. If you determine it needs more discussion, just ping me on the community spinnaker slack or kindly open a new issue.
